c7n/manager.py

```diff
@@ -122,7 +122,7 @@ def filter_resources(self, resources, event=None):
 
             if event and event.get('debug', False):
                 self.log.debug(
-                    "applied filter %s %d->%d", f, rcount, len(resources))
+                    "applied filter %s %d->%d", f.data, rcount, len(resources))
```

Comment:
instead of inlining the filter content into the line how about add it as a separate block after the count, something along the lines of
"filtered resources %d -> %d filter:\n %s", rcount, len(resources), f.data)

Reply:
yeah. i was thinking about that... except not adding a newline.  that way it's easier to read the effect it had (how many resources were filtered), but the why is still right there.  And unless we pretty print the whole block, personally it doesn't make sense to me to add the newline.  AWS's console can pretty print a trailing json block it finds for you in cloudwatch, so maybe we need to just make sure it's printed in a way compatible with that?

Reply:
Is there a reason `c7n.utils.dumps` defaults to `indent=0` which causes newlines to be introduced?  shouldn't that default to `indent=None` so it's all on one line and only pretty print it if indent is specified?

Reply:
default is to ensure readability, with reporting cli for for additional ux and format conversion..  in this context, simply passing the param explicit seems like it addresses.  looking through machine readable aspects, we need a different implementation (jsonlines) all together for some of the use cases on object storage query.